docs/tutorials/Notebooks.md

```diff
@@ -3,10 +3,10 @@
 We provide Google Colab-based notebook packaging for a quickstart to
 OpenROAD-flow-scripts as follows:
```

Comment:
Add this a benefit to using Colab: You can collaborate with your team and easily manage  your OpenROAD design project runs using the Google Colab notebook over the cloud. This helps you to make interactive changes to python or tcl scripts and re-run the flow to examine the impact the changes all from the Notebook.